src/App.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import 'bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
This is repeated in `index.js`. Can we just keep this stuff and make `index.js` pure of CSS?